deploy/helms/legion/templates/edge/edge-rbac.yaml

```diff
@@ -7,6 +7,13 @@ rules:
 - apiGroups: [""] # core API group
   resources: ["services"]
   verbs: ["watch", "list"]
+- apiGroups: [""] # core API group
+  resources: ["configmaps"]
+  verbs: ["create", "get", "patch", "update"]
+  resourceNames: ["{{ .Release.Name }}-invalid-tokens"]
```

Comment:
@aliaksandr-d  [minor] invalid-tokens ->blacklisted-tokens for all places in this PR where "invalid-tokens" is used